atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueService.java

```diff
@@ -1642,9 +1642,18 @@ private void putMetadataForTablesInternal(
     }
 
     private void putMetadataWithoutChangingSettings(final TableReference tableRef, final byte[] meta) {
```

Comment:
Would of course also be good to have testing for this.  If you don't have time I can write some before we commit.